test/syscalls/linux/tuntap.cc

```diff
@@ -490,5 +491,36 @@ TEST_F(TuntapTest, WriteHangBug155928773) {
   write(sock, "hello", 5);
 }
 
+// EmptyInterfaceName tests empty interface name at creation of tun/tap.
+TEST_F(TuntapTest, EmptyInterfaceName) {
+  SKIP_IF(!ASSERT_NO_ERRNO_AND_VALUE(HaveCapability(CAP_NET_ADMIN)));
+
+  FileDescriptor sender = ASSERT_NO_ERRNO_AND_VALUE(
+      Socket(AF_INET, SOCK_STREAM | SOCK_NONBLOCK, IPPROTO_TCP));
+
+  char* empty_name = "";
+  struct ifreq ifr_set = {};
+
+  // Interface creation.
+  ASSIGN_OR_RETURN_ERRNO(FileDescriptor fd, Open(kDevNetTun, O_RDWR));
+
+  std::memset(&ifr_set, 0x0, sizeof(ifr_set));
```

Comment:
`ifr_set` is already zeroed, so this can be deleted.

Reply:
Fixed.